.github/workflows/release.yml

```
@@ -73,15 +73,15 @@ jobs:
-          cd flipper_models
-          flutter test --dart-define=FLUTTER_TEST_ENV=true
+          flutter test --dart-define=FLUTTER_TEST_ENV=true --verbose
+          # cd ..
+          # cd flipper_models
+          # flutter test --dart-define=FLUTTER_TEST_ENV=true --verbose
 
   integration-testing-windows:
     name: "Integration Testing Windows"
     runs-on: windows-2022
-    needs: [unit-testing,android-integration-test]
+    needs: [unit-testing,integration-on-android]
```

YAMLlint (1.35.1)

[warning] too few spaces after comma

(commas)